mmhuman3d/data/data_converters/agora.py

```diff
@@ -40,6 +41,79 @@ def __init__(self, modes: List = [], fit: str = 'smpl',
                 Use either (1280, 720) or (3840, 2160)')
         self.res = res
 
+    def get_global_orient(self,
+                          imgPath,
+                          df,
+                          i,
+                          pNum,
+                          globalOrient=None,
+                          meanPose=False):
+
+        if 'hdri' in imgPath:
+            camYaw = 0
```

Comment:
Are these hard-coded values official? It would be very helpful to indicate in the comments the references we take to obtain these values, it can be the paper sections or official implementation codes.

Reply:
added in [95e71cc](https://github.com/open-mmlab/mmhuman3d/pull/143/commits/95e71ccf18678d0b7f0a60b5322380785e37eff1)